quickcache/quickcache_helper.py

```diff
@@ -122,6 +124,12 @@ def _serialize_for_key(self, value):
         elif isinstance(value, set):
             return 'S' + self._hash(
                 ','.join(sorted(map(self._serialize_for_key, value))))
+        elif isinstance(value, datetime.datetime):
```

Comment:
I might insert a code comment here just saying

```
# Cache key equality for datetimes follows python equality. Namely:
# - Datetimes with different timezones but representing the same point in time are serialized the same way
# - Naive datetimes can't cause a cache hit for tz aware datetimes (and vice versa)
```

Was confusing enough a decision that I think it's worth leaving both the what (the bullet points) and the why (that we're imitating python's datetime equality).

Reply:
right on makes sense